crates/spidermonkey-embedding-splicer/src/stub_wasi.rs

```diff
@@ -369,10 +368,10 @@ fn stub_stdio(module: &mut Module) -> Result<()> {
     })?;
 
     // (func (param i32 i32 i32 i32) (result i32)))
-    let len_local = module.locals.add(ValType::I32);
     stub_import(module, PREVIEW1, "fd_write", |body| {
-        body.local_get(len_local);
-        Ok(vec![len_local])
+        let len_local: u32 = 3; // Index of the last local
```

Comment:
Would it make sense to rather write:

```suggestion
        let len_local = LocalID(3); // Index of the last local
```

and use this below?